### DISCUSSION TITLE
A simple thank you for a well-designed React plugin

### DISCUSSION THREAD

**Discussion by joshuaiz** [igordanchenko/yet-another-react-lightbox#154] [General]
Hello,

Just wanted to say thank you for this plugin — I had it installed and working in minutes. The docs and examples are clear which makes using this a breeze.

The best part: *it just works*: no unnecessary context or app wrapping just add the component + slides and you are good to go.

**Comment by igordanchenko** [comment]
Thank you for your feedback!
